### ISSUE TITLE
Autoupdate on post feed

### ISSUE THREAD

**Issue by makise-homura** [makise-homura/kurisaba#11]
Post feed should have autoupdate, as common threads have.

**Comment by mamoruarai** [comment]
Автообновление должно включаться и выключаться по желанию.

**Comment by makise-homura** [comment]
Хм, оказалось сильно сложнее, чем я думал - там надо не просто заменять всю таблицу постов, там, похоже, надо будет ручками проверять идентичность каждого postnode и его карты ответов, и менять в зависимости от этого. А то ломается вся динамика на странице (например, скрытие постов и превьюшки ютуба).
Ладно, мб в понедельник покопаюсь или как найду время.

**Comment by mamoruarai** [comment]
Пожалуйста, сделай так, чтобы однопоток обновлялся не вместе с плавающей формой. Чтобы там постить можно было.

**Comment by makise-homura** [comment]
Да, это следствие обновления всего div-а с постами, вместо содержимого каждого postnode. Должно будет починиться, когда я сделаю по-нормальному.
